MIP4/mip4.md

```diff
@@ -48,8 +50,13 @@ The motivation behind this proposal is that changing small details to MIPs shoul
 **Amendments**
 MIP Amendments that preserve the MIP number can be performed as long as there are no changes to the logic of the MIP or to the MIP's external output dependencies. They should only be used when minor changes are required.
 
+Amendments to MIPs under 3 months old follow a more flexible set of rules outlined in c2.
+
+Amendments to multiple MIPs are allows to be submitted as a single proposal.
```

Comment:
allows -> allowed.

Reply:
Also, there needs to be more detail here. Multiple MIPs should only be allowed to be amended by the same amendment sub-proposal if the changes are linked in some way. IE: Same MIP Set, part of a larger change, etc.